.github/workflows/build_publish.yml

```diff
@@ -43,7 +40,30 @@ jobs:
         run: |
           sudo gem install compass -v 1.0.3
       - name: Build and deploy with Maven
-        run: ./mvnw --no-transfer-progress clean -U deploy
+        run: ./mvnw --no-transfer-progress clean -U package -DskipTests
+        
+      - name: List files in target directory
+        run: ls -al bahmnicore-omod/target
```

Comment:
@karimshaik786 is this step required in the workflow ? if yes, can you describe the significance